plugins/frontend/backstage-plugin-iframe/src/components/IFrameComponent.tsx

```diff
@@ -28,11 +30,23 @@ import { determineError } from './utils/helpers';
  */
 export const IFrameCard = (props: IFrameProps) => {
   const { src, height, width } = props;
+  const { entity } = useEntity();
+
   const title =
     props.title || 'Backstage IFrame (Note you can modify this with the props)';
   const configApi = useApi(configApiRef);
   const allowList = configApi.getOptionalStringArray('iframe.allowList');
   const errorMessage = determineError(src, allowList);
+  let sanitizedSrc = '';
+
+  try {
+    // In theory this might be extended to include the logged in user maybe.
+    const evaluatedSrc = renderString(src, { entity });
+    // The following attempts to sanitize the url before sending it to the iframe.
+    sanitizedSrc = new URL(evaluatedSrc).toString();
+  } catch (e) {
+    // pass
```

Comment:
probably good idea to add a fallback or an error msg rendering here instead of silent failure. You can probably use a boolean useState hook that can be then used to conditionally render error view if the URL is borked

Reply:
done